diff -urN Catalyst-Authentication-Credential-OpenID-0.16/Makefile.PL Catalyst-Authentication-Credential-OpenID-0.16-patched/Makefile.PL
--- Catalyst-Authentication-Credential-OpenID-0.16/Makefile.PL	2009-10-14 01:38:02.000000000 +1100
+++ Catalyst-Authentication-Credential-OpenID-0.16-patched/Makefile.PL	2012-10-25 02:45:04.283029013 +1100
@@ -13,8 +13,6 @@
 requires "Catalyst::Devel"    => "1";
 requires "Net::OpenID::Consumer" => "1.03";
 requires "Catalyst::Authentication::User::Hash" => undef;
-requires "Catalyst::Plugin::Session::Store::FastMmap" => "0.05";
-requires "Catalyst::Plugin::Session::State::Cookie" => "0.08";
 requires "Catalyst::Engine::HTTP" => undef;
 
 recommends "Config::General"   => "2";
